package co.ke.xently.features.auth.data.domain.error

data object UnknownError : Error